comtypes/client/_events.py

```diff
@@ -222,12 +222,12 @@ def __getattr__(self, name):
         "Create event handler methods on demand"
         if name.startswith("__") and name.endswith("__"):
             raise AttributeError(name)
-        print("# event found:", name)
+        print(f"# event found: {name}")
```

Comment:
This part is not related to the issue; rather, it's where spaces are inserted due to the default `sep` of [`print`](https://docs.python.org/3/library/functions.html#print).

Please revert this line to its original.